tenablesc.json

```
@@ -76,45 +76,70 @@
             "identifier": "scan_endpoint",
             "read_only": true,
             "parameters": {
+                "scan_name": {
+                    "description": "Title of the Tenable scan",
+                    "data_type": "string",
+                    "required": true,
+                    "order": 0
+                },
```

Adding new required parameters is likely a breaking change for existing automation. Let's make it not required and give it a default?
```suggestion
                "scan_name": {
                    "default": "Scan launched from Splunk SOAR",
                    "description": "Title of the Tenable scan",
                    "data_type": "string",
                    "order": 0
                },